models/POP/shell_scripts/cesm2_1/HPSS_get_ensemble

```diff
@@ -0,0 +1,67 @@
+#!/bin/csh -f
+#
+# Copyright 2020 University Corporation for Atmospheric Research
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License.
+# Please view the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# ==============================================================================
+# 
+# ---------------------
+# Purpose
+# ---------------------
+#
+# This script gets 80 POP restart files from HPSS.
+#
+# ==============================================================================
```

Comment:
No it doesn't ... 270-209 + 1 = 62 files.
Since you have  a 'year_span' ... are you trying to support getting separated years to use as an initial condition?
If that is the case, you could mention that in the Purpose, and I'd suggest using 'year_stride' rather than 'year_span'